Misc/NEWS

```diff
@@ -10,6 +10,8 @@ What's New in Python 3.6.1 release candidate 1?
 Core and Builtins
 -----------------
 
+- bpo-29438: Fixed use-after-free problem in key sharing dict.
```

Comment:
I'm not sure about "bpo-" in Misc/NEWS and commit message, but if this is correct the patch LGTM.